### PR TITLE
Util

### PR DESCRIPTION
**Pull request by kanake10**

### CODE REVIEW

File: core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @Test
+    fun `Should Check If Palindrome`() {
+        assertTrue("radar".isPalindrome())
+        assertTrue("A man a plan a canal Panama".isPalindrome())
+    }
+}
+
+
+
+
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt:47]: Just a few extra new lines at the end here.
```suggestion

```

File: core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+object Utils {
+    @JvmStatic
+    fun foo(): Boolean {
+        return false
+    }
+}
+
+
+class  UtilTests{
```
Comment (KevinGilmore) [inline core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt:31]:
```suggestion
class UtilTests {
```

Comment (KevinGilmore) [reply core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt:31]: Let's fix the spacing in this line.

File: core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import org.junit.Test
+
+
+object MathUtils{
+    fun square(number: Int): Int {
+        return number * number
+    }
+}
+
+object ExtensionUtil{
```
Comment (KevinGilmore) [inline core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt:16]:
```suggestion
object ExtensionUtil {
```

File: core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt [hunk 4]
```diff
@@ -0,0 +1,43 @@
+package com.baeldung.util
+
+import com.baeldung.util.ExtensionUtil.isPalindrome
+import com.baeldung.util.MathUtils.square
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+
+object MathUtils{
```
Comment (KevinGilmore) [inline core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/util/Utils.kt:10]:
```suggestion
object MathUtils {
```

**Pull request merged**